src/Layout.tsx

```diff
@@ -4,25 +4,55 @@ import { nanoid } from 'nanoid'
 import randomWords from 'random-words'
 import { configureAbly } from '@ably-labs/react-hooks'
 import InfoCard from './InfoCard'
-
+import { Types } from 'ably'
+import { SignJWT } from 'jose'
 const clientId = nanoid()
 const example: string = window.location.pathname
-let API_KEY: string | undefined
-
+let API_CONFIG: Types.ClientOptions = { clientId }
 switch (example) {
   case '/avatar-stack':
-    API_KEY = import.meta.env.VITE_ABLY_KEY_AVATAR_STACK
+    API_CONFIG.key = import.meta.env.VITE_ABLY_KEY_AVATAR_STACK
```

Comment:
This is cool - can you please add some explanation about this in the main (repo) README as well, please?

Reply:
I updated the readme and this code to allow for a specific API key for the claims example inline with the other two examples (https://github.com/ably-labs/realtime-examples/pull/37/commits/b3370c735250360568c2b49953cd9f5b8b352b12), but I'm not sure what else you'd like me to add to the readme about this part

Reply:
These are the things I was hoping to get documented in the main README, so users (& I!) know what we're importing and why:
- `import { Types } from 'ably'`
- `import { SignJWT } from 'jose'`


Happy for these to have inline comments rather than updating the README.